src/build-2024/docs/Introduction.md

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
I assume this is still a work in-progress along with the other overview articles.

Be sure to add a step in one of these to set the environment variable that will be used for the samples.

Reply:
It is listed [here](https://github.com/zewditu/ai-samples/blob/U/zewd/Build2024/src/build-2024/README.md) as one of the prerequisites.